static/js/unread.js

```diff
@@ -573,10 +618,19 @@ export function num_unread_for_topic(stream_id, topic_name) {
     return unread_topic_counter.get(stream_id, topic_name);
 }
 
+export function stream_has_any_unread_mentions(stream_id) {
+    const streams_with_mentions = unread_topic_counter.get_streams_with_unread_mentions();
+    return streams_with_mentions.has(stream_id);
+}
+
```

Comment:
Added this separate function to just return the info about `unread_mentions` being present or not for a `stream_id` without much calculation;